### PR TITLE
readme-description added

### PR DESCRIPTION
**Pull request by ferhatu**

### CODE REVIEW

**Review by minwwgo — COMMENTED** [review]
everything is good, keep go on

**Comment by 279Roxana** [comment]
Great job!

**Review by ahmad-ali14 — APPROVED** [review]
gode job

**Review by minwwgo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
 
 # Well Minded
 The main goal of the project is to create a space where people can get information about free support towards their mental health. The platform is meant to gather charities, free support groups, and free resources in the same place as at the moment, finding accessible professional support can be challenging. This fact may be discouraging for people who are already struggling and are trying to find alternative professional support,  as there are many users who have tried clinical settings and are looking for a therapeutic safe space. The platform can be used by people dealing with mental health issues or people from their support network. As the website will gather organisations, users will be redirected to the contact of these.
+
```
Comment (ahmad-ali14) [inline README.md:4]: good

**Review by 137rx — APPROVED** [review]

**Review by ahmad-ali14 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ferhatu) [reply README.md:4]: thank you Ahmad

**Pull request merged**